pkg/skaffold/jib/jib.go

```diff
@@ -60,14 +72,24 @@ func getDependencies(cmd *exec.Cmd) ([]string, error) {
 		}
 
 		if !info.IsDir() {
+			// try to relativize the path
+			if relative, err := relativize(dep, workspace, canonicalWorkspace); err == nil {
```

Comment:
I wonder if `dep` is most of the time canonical here. If so, you could swap `workspace` and `canonicalWorkspace` so that the good case comes first.

Reply:
Good idea.